src/views/SystemIntake/ContractDetails/index.tsx

```diff
@@ -165,57 +165,76 @@ const ContractDetails = ({
+                              id="IntakeForm-FundingSource"
+                              name="fundingSource.source"
+                            >
                               <Field
                                 as={DropdownItem}
-                                key={source.split(' ').join('-')}
-                                name={source}
-                                value={source}
+                                name="Select an option"
+                                value=""
+                                disabled
                               />
-                            ))}
-                          </Field>
-                        </FieldGroup>
-                        <FieldGroup
-                          scrollElement="fundingSource.fundingNumber"
-                          error={!!flatErrors['fundingSource.fundingNumber']}
-                        >
-                          <Label htmlFor="IntakeForm-FundingNumber">
-                            Funding Number
-                          </Label>
-                          <FieldErrorMsg>
-                            {flatErrors['fundingSource.fundingNumber']}
-                          </FieldErrorMsg>
-                          <Field
-                            as={TextField}
+                              {fundingSources.map(source => (
+                                <Field
+                                  as={DropdownItem}
+                                  key={source.split(' ').join('-')}
+                                  name={source}
+                                  value={source}
+                                />
+                              ))}
+                            </Field>
+                          </FieldGroup>
+                          <FieldGroup
+                            scrollElement="fundingSource.fundingNumber"
                             error={!!flatErrors['fundingSource.fundingNumber']}
-                            id="IntakeForm-FundingNumber"
-                            maxLength={6}
-                            name="fundingSource.fundingNumber"
-                          />
-                        </FieldGroup>
+                          >
+                            <Label htmlFor="IntakeForm-FundingNumber">
+                              Funding Number
+                            </Label>
+                            <FieldErrorMsg>
+                              {flatErrors['fundingSource.fundingNumber']}
+                            </FieldErrorMsg>
+                            <Field
+                              as={TextField}
+                              error={
+                                !!flatErrors['fundingSource.fundingNumber']
+                              }
+                              id="IntakeForm-FundingNumber"
+                              maxLength={6}
+                              name="fundingSource.fundingNumber"
```

Comment:
We need an `aria-describedby="IntakeForm-FundingNumberHelp"` here so that the external link is associated with this field.